src/App.jsx

```diff
@@ -1,80 +1,15 @@
 import React from 'react';
-import './App.css';
-// Move each BEM block to a separate component (file) and import them here
+import './App.scss';
```

Comment:
```suggestion
import './App.css';
```
![image](https://user-images.githubusercontent.com/96409563/232279143-90848b37-4cfc-4679-b7e3-bf31110e4d94.png)